nucypher/acumen/perception.py

```diff
@@ -30,146 +33,323 @@
 from nucypher.utilities.logging import Logger
 
 
+class BaseFleetState:
+
```

Comment:
There's a somewhat different behavior for `nickname` in `FleetState` and `ArchivedFleetState`, and increasing complexity just because of `checksum` didn't seem to worth it. But I agree, currently `BaseFleetState` is more of a mixin adding `__str__` behavior.

Reply:
Removed this completely. `ArchivedFleetState` doesn't seem to need `__str__` anyway.